### DISCUSSION TITLE
How to install connectkit package locally for development

### DISCUSSION THREAD

**Discussion by aschenkel** [family/connectkit#389] [Q&A] [answered]
I want to work on this connectkit [open issue](https://github.com/family/connectkit/issues/379).

What are the instructions I should follow in order to test my app with a local connectkit package while updating it?

**Comment by lochie** [comment, marked as answer]
thanks for wanting to contribute @aschenkel!

our readme shows how to set up the local examples:
https://github.com/family/connectkit?tab=readme-ov-file#running-examples-locally

though you may need to run `yarn build` before `yarn dev:connectkit` will work
we have a testbench with a lot of functionality you can use by running `yarn dev:testbench` along with `yarn dev:connectkit`

if you want to test on your own app running in a different project, you can target a local package via `file:` in your package.json after you've run `yarn build` command in the connectkit project.
example: `"connectkit": "file:../connectkit/packages/connectkit"`

**Reply by aschenkel** [reply]
ok, I was missing that `yarn build` step. I'll test that later today.

Thanks!
